.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,18 @@
+#### [LEVEL]/[COMPONENT]
+| TASK | <!--- [jira TASK ID](jiraURL) -->             |
+| ---- | --------------------------------------------- |
+| DEMO | <!--- [[LEVEL]/[COMPONENT]](zeit now URL) --> |
+
+##### Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+##### Description, Motivation and Context
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
```

Comment:
what about asking for how to reproduce in case it's a bugfix?

Reply:
<!--- If it fixes an open issue, please link to the issue here. --> is removed, sorry

Reply:
Ok, I'll add it too